content/inference/particlenet.md

```diff
@@ -993,76 +1095,77 @@ Then, we show three NN model configurations below and provide detailed explanati
 
-Here we present three ways of training. For readers who have a local machine with CUDA GPUs, please try out training on the local GPUs. Readers who would like to try on CPUs can also refer to the local GPU instruction. It is also possible to borrow the GPU resources from the lxplus condor or CMS-connect. Please find in the following that meets your situation.
+Here we present three ways of training. For readers who have a local machine with CUDA GPUs, please try out training on the local GPUs. Readers who would like to try on CPUs can also refer to the local GPU instruction. It is also possible to borrow the GPU resources from the lxplus HTCondor or CMS Connect. Please find in the following that meets your situation.
 
 === "Train on local GPUs"
 
-    The three networks can be trained with a universal script. Enter the `weaver` base folder and run the following command. Note that `${data_config}`, `${model_config}`, and `${prefix}` refers to the value in the above table for each example, and the fake path should be replaced with the correct one.
+    The three networks can be trained with a universal script. Enter the `weaver` base folder and run the following command. Note that `${DATA_CONFIG}`, `${MODEL_CONFIG}`, and `${PREFIX}` refers to the value in the above table for each example, and the fake path should be replaced with the correct one.
 
     ```python
+    PREFIX='<prefix-from-table>'
+    MODEL_CONFIG='<model-config-from-table>'
+    DATA_CONFIG='<data-config-from-table>'
+    PATH_TO_SAMPLES='<your-path-to-samples>'
+
     python train.py \
-    --data-train '<path-to-samples>/prep/top_train_*.awkd' \
-    --data-val '<path-to-samples>/prep/top_val_*.awkd' \
-    --fetch-by-file --fetch-step 1 \
-    --num-workers 3 \
-    --data-config data/benchmark/${data_config} \
-    --network-config networks/benchmark/${model_config} \
-    --model-prefix output/${prefix} \
-    --gpus 0,1 --batch-size 1024 --start-lr 5e-3 --num-epochs 20 --optimizer ranger \
-    --log output/${prefix}.train.log
+     --data-train ${PATH_TO_SAMPLES}'/prep/top_train_*.root' \
+     --data-val ${PATH_TO_SAMPLES}'/prep/top_val_*.root' \
+     --fetch-by-file --fetch-step 1 --num-workers 3 \
+     --data-config top_tagging/data/${DATA_CONFIG} \
+     --network-config top_tagging/networks/${MODEL_CONFIG} \
+     --model-prefix output/${PREFIX} \
+     --gpus 0,1 --batch-size 1024 --start-lr 5e-3 --num-epochs 20 --optimizer ranger \
+     --log output/${PREFIX}.train.log
     ```
 
-    Here are several caveats running this command.
+    Here `--gpus 0,1` specifies the GPUs to run with the device ID 1 and 2. For training on CPUs, please use `--gpu ''`.
+    
+    A detailed description of the training command can be found in [`Weaver` README](https://github.com/hqucms/weaver). We would say a few more words in the data loading options, because it depends on the scenarios we feed in the input data. Here are several caveats.
```

Comment:
Below we will note a few more caveats about the data loading options, though the specific settings will depend on the specifics of the input data.